superset-frontend/cypress-base/cypress/integration/dashboard/dashboard.helper.ts

```diff
@@ -45,7 +45,9 @@ export interface ChartSpec {
 
 export function getChartGridComponent({ name, viz }: ChartSpec) {
   return cy
-    .get(`[data-test="chart-grid-component"][data-test-chart-name="${name}"]`)
+    .get(`[data-test="chart-grid-component"][data-test-chart-name="${name}"]`, {
+      timeout: 30000,
+    })
```

Comment:
This was just resolving very close to the default 5 secs, and apparently started causing widespread flakiness on cypress. I bumped this to the 30s seen a few lines below, now all tests pass.